### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by chesn0k**

### CODE REVIEW

File: tests/functional/Generator/_phpstorm_meta/.phpstorm.meta.php/database.php [hunk 1]
```diff
@@ -55,7 +55,6 @@
     'search_index',
     'search_total',
     'sequences',
-    'sessions',
```
Comment (chesn0k) [inline tests/functional/Generator/_phpstorm_meta/.phpstorm.meta.php/database.php:58]: https://www.drupal.org/project/drupal/issues/3428565

File: tests/functional/Helper/Drupal/ServiceInfoTest.php [hunk 2]
```diff
@@ -94,7 +94,6 @@ public function testGetServiceDefinitions(): void {
                 'invalidBehavior' => 1,
               ],
             ],
-          'resolve' => TRUE,
```
Comment (chesn0k) [inline tests/functional/Helper/Drupal/ServiceInfoTest.php:97]: https://www.drupal.org/project/drupal/issues/3432738

File: tests/functional/Helper/Drupal/ServiceInfoTest.php [hunk 3]
```diff
@@ -147,7 +146,6 @@ public function testGetServiceDefinition(): void {
...
```
Comment (chesn0k) [inline tests/functional/Helper/Drupal/ServiceInfoTest.php:150]: https://www.drupal.org/project/drupal/issues/3432738

**Comment by chesn0k** [comment]
https://www.drupal.org/project/drupal/issues/3343369

~looks like we'll have to remove the action plugin generator.~

**Comment by Chi-teck** [comment]
> https://www.drupal.org/project/drupal/issues/3343369
> 
> looks like we'll have to remove the action plugin generator.

I think that about Action UI

**Comment by chesn0k** [comment]
> > https://www.drupal.org/project/drupal/issues/3343369
> > looks like we'll have to remove the action plugin generator.
> 
> I think that about Action UI

Yes, you're right, this only applies to UI

**Comment by Chi-teck** [comment]
> > > https://www.drupal.org/project/drupal/issues/3343369
> > > looks like we'll have to remove the action plugin generator.
> > 
> > 
> > I think that about Action UI
> 
> Yes, you're right, this only applies to UI

If we can't test it without this module, I am fine to drop this generator. Seems nobody besides me is using it.

File: tests/sut/qux/tests/src/Kernel/ActionTest.php [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    self::assertEquals('Update node field', $action->getPluginDefinition()['label']);
+    self::assertEquals('DCG', $action->getPluginDefinition()['category']);
+    self::assertEquals('node', $action->getPluginDefinition()['type']);
+
+    $node = $this->createNode(['type' => 'article']);
+    self::assertInstanceOf(AccessResultInterface::class, $action->access($node, return_as_object: TRUE));
+
+    $action->execute($node);
+    $field_example_value = $node->get('field_example')->first()?->get('value')->getValue();
+    self::assertEquals('New value', $field_example_value);
```
Comment (chesn0k) [inline tests/sut/qux/tests/src/Kernel/ActionTest.php:73]: Not sure about the effectiveness of this part of the test

**Comment by Chi-teck** [comment]
@chesn0k there is a new portion of changes in Drupal core that breaks our tests. Can you check it?

**Pull request merged**